frontend/pages/index.tsx

```diff
@@ -3,20 +3,36 @@ import { GetServerSideProps } from "next";
 import LoginLogoutButton from "@frontend/components/login-logout-button";
 import { Box, Img, Flex, Heading } from "@chakra-ui/react";
 import Dashboard from "@frontend/components/homepage/dashboard";
+import { USER_ROLES } from "@frontend/utils/constants";
+import { serverURL } from "@frontend/config";
+import { StatusDataProps } from "@frontend/models/patient";
 
 export const getServerSideProps: GetServerSideProps = async context => {
+    let session = await getSession(context);
+    let userId = session?.user.AccountId;
+    let role = session?.user.Role;
+    let data: any[] = [];
+
+    if (role === USER_ROLES.patient) {
+        const response = await fetch(serverURL + "/status/getAllStatus/" + userId);
+        data = await response.json();
+    }
+
     return {
         props: {
             session: await getSession(context),
+            data,
         },
     };
 };
 
-export default function Home() {
-    const { data: session } = useSession();
+export default function Home({ data }: {data: StatusDataProps[]}) {
+    console.log( data);
```

Comment:
console log("remove this console log please");

Reply:
xDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD